support/metrics/metricsTracker.go

```diff
@@ -196,28 +244,28 @@ func (mt *MetricsTracker) GetUpdateEventSentTime() *time.Time {
 }
 
 // SendStartupEvent sends the startup Amplitude event.
-func (mt *MetricsTracker) SendStartupEvent() error {
-	return mt.sendEvent(startupEventName, mt.props)
+func (mt *MetricsTracker) SendStartupEvent(now time.Time) error {
+	return mt.SendEvent(startupEventName, mt.props, now)
 }
 
 // SendUpdateEvent sends the update Amplitude event.
 func (mt *MetricsTracker) SendUpdateEvent(now time.Time, success bool, millisForUpdate int64) error {
-	commonProps := mt.props
-	commonProps.SecondsSinceStart = now.Sub(mt.botStartTime).Seconds()
+	secondsSinceStart := now.Sub(mt.botStartTime).Seconds()
```

Comment:
we don't need this here, can put into the if condition where updateEventSentTime is nil since we don't set it explicitly on the commonProps anymore